pynapple/io/neurosuite.py

```diff
@@ -372,6 +372,35 @@ def load_lfp(self, filename=None, channel=None, extension='.eeg', frequency=1250
                 time_support = time_support,
                 columns=channel)
 
+            
+    def read_neuroscope_intervals(self, name):
+        """
+        This function reads .evt files in which odd raws indicate the beginning 
+        of the time series and the even raws are the ends. If the file is not present
+        in the nwb, it loads the events from the nwb directory.
+        
+    
+        Parameters
+        ----------
+        name: name of the .evt file
+    
+        Returns
+        -------
+        IntervalSet  
+    
+        """
+        isets = self.load_nwb_intervals(name)
+        if str(type(isets)) !="<class 'pynapple.core.interval_set.IntervalSet'>":
+            print("reading intervals...")
+            evt_file = os.path.join(self.path, self.path.split('/')[-1] + "." + name + ".evt")
+            df = pd.read_csv(evt_file, delimiter=' ', usecols = [0], header = None)
+            start = nap.Ts(df.iloc[::2].values, time_units='ms')
```

Comment:
It's too tedious. I would just reshape df.values with 2 columns and call intervalSet. 
Also it's possible that some files contains a start time, a middle time and an end time. I don't think we can assume it's always start and end.

Reply:
Changed